test/runtests.jl

```diff
@@ -1,5 +1,5 @@
 # Build the real docs first.
-include("../docs/make.jl")
+# include("../docs/make.jl")
```

Comment:
I think it is two problems. Doc building and deprecations in stuff needed for coverage.